ZSRPHB/SudokuTable.cs

```diff
@@ -0,0 +1,332 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.Json;
+
+
+    internal static class SudokuTable

```

Comment:
Ezeknek nagy része lehetett volna extension method is, de állapotot tárol is, szóval én nem tettem volna `static` osztállyá ezt, mert ezt aztán tesztelni nehéz lenne a jövőben.